substream.js

```diff
@@ -68,6 +68,24 @@ function factory(Stream) {
   SubStream.OPEN    = 3;   // The connection is open.
 
   /**
+   * wrap emit event to emit events across the wire
+   *
+   * @param  {String} event
+   * @param  {*} ...args
+   */
+  SubStream.prototype.emit = function (event/* ..args */) {
```

Comment:
It seems that `emit` now writes on the underlying connection but also emit the event to the instance it is called from. It looks wrong to me.